### ISSUE TITLE
Quirk customization poppers are incorrectly positioned, and overlap onto the button

### ISSUE THREAD

**Issue by nikothedude** [NovaSector/NovaSector#347]
Reporting client version: 515.1609

## Issue Summary

Title. The popper is too high, and makes it impossible to hit the dedicated "close the quirk customziation" button.

## Round ID:
10448

## Testmerges:
- [Antag optin ](https://github.com/NovaSector/NovaSector/pull/121)
- [[TM ONLY] [EARLY MIRROR] Revert "Makes immerse use weakrefs", prioritizing some ref removal lines instead](https://github.com/NovaSector/NovaSector/pull/238)
- [[MODULAR] Removes Arbitrary Loadout Restrictions](https://github.com/NovaSector/NovaSector/pull/58)
- [[MODULAR] Chat color tweaks!](https://github.com/NovaSector/NovaSector/pull/323)
- [Kahraman, after collecting dust for 100 years, finally has its own content, hear ye guncargo naysayers and weep](https://github.com/NovaSector/NovaSector/pull/98)

## Reproduction:

Step 1: Get a customizable quirk
Step 2: Open the quirk customization
Step 3:
![image](https://github.com/NovaSector/NovaSector/assets/59709059/02b7c06c-73c1-4e2d-890f-9dd5f2374aab)

**Comment by Floofies** [comment]
While still an issue, it is not a very impactful one. Players are still able to click anywhere else on the page to close the popup.

**Comment by nikothedude** [comment]
> While still an issue, it is not a very impactful one. Players are still able to click anywhere else on the page to close the popup.

false, it prevents people from reading the tooltip below the popper for quirk customization

very important for death degradation

**Comment by nikothedude** [comment]
i made a comment but it was for the wrong issue report
